types/sync-storage/index.d.ts

```diff
@@ -0,0 +1,26 @@
+// Type definitions for sync-storage 0.4
+// Project: https://github.com/raphaelpor/sync-storage
+// Definitions by: Dannomac <https://github.com/Dannomac>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/*
+*  Init prepares the SyncStorage to work synchronously, by getting all values for all keys previously saved on RN AsyncStorage.
+*/
+export function init(): any;
```

Comment:
The return type should probably be `void` here

Reply:
Your right. I have updated this in my latest commit.